derp/derphttp/derphttp_server.go

```diff
@@ -20,6 +20,10 @@ const fastStartHeader = "Derp-Fast-Start"
 
 func Handler(s *derp.Server) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.Method == http.MethodGet && r.RequestURI == "/debug/traffic" {
```

Comment:
Delete. This will only be in cmd/derper.

Also, this will never be routed, as this handler is installed at path /derp, so /debug/* can't get here.

Also, install the route unconditional on method. The handler itself can check the method and fail on bogus ones.